src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/SubscriptionServiceImpl.java

```diff
@@ -115,19 +114,21 @@ public void collectSubscribers(Artefact artefact) {
         artefact.getSearch().get("cases").forEach(object -> subscriptionList.addAll(extractSearchValue(object)));
 
         List<Subscription> subscriptionsToEmail;
-        if (artefact.getSensitivity().equals(Sensitivity.CLASSIFIED)) {
-            subscriptionsToEmail = validateSubscriptionPermissions(subscriptionList, artefact.getListType());
+        if (artefact.getSensitivity().equals(Sensitivity.CLASSIFIED)
+            || artefact.getSensitivity().equals(Sensitivity.PRIVATE)) {
```

Comment:
do we need to check for private? anyone who can subscribe can see private so would this be needed?

Reply:
I've now removed this :)